alpha/scripts/analyzeEmailFieldForPartner.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+require_once(__DIR__ . '/bootstrap.php');
+
+// parse the command line
+if($argc<3)
+{
+	die("Usage: php " . $argv[0] . " <partner id> <user type: admin | user> <realRun | dryRun>\n");
+}
+
+$partnerId = $argv[1];
+$userType = $argv[2];
+$dryRun = $argv[3] ? $argv[3] : 'dryRun';
+var_dump($partnerId, $userType, $dryRun);
+
+if (!PartnerPeer::retrieveByPK($partnerId))
+{
+	die("Please enter a valid partner Id!\n");
+}
+
+if (is_null($userType) || ($userType != 'admin' && $userType != 'user'))
+{
+	die("Please specify if looking for admin users (admin) or regular users (user)\n");
+}
+
+$isAdmin = 0;
+if ($userType === 'admin')
+{
+	$isAdmin = 1;
+}
+
+
+
+function countUsers($partnerId, $isAdmin, $emailFieldValue)
```

Comment:
i would have just change the variable to be $hasEmail or something like that with true/false and not pass and compare to criteria